.eslintrc

```diff
@@ -57,5 +57,8 @@
       "error",
       { "props": true, "ignorePropertyModificationsFor": ["state"] }
     ]
+  },
+  "globals": {
+    "JSX": "readonly"
```

Comment:
question: what's this for?

Reply:
to stop eslint from complaining no-undef warning for `jsx`

Reply:
To remove warning of undefined `JSX` in lint